Documentation/data-collection.md

```diff
@@ -9,55 +9,55 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
 apiVersion: v1
 data:
   metrics.yaml: |-
-    "matches":
-    - "{__name__=~\"cluster:usage:.*\"}"
```

Comment:
Do you mind explaining why these changes?

Reply:
they are just readability, I removed the surrounding quotes and the unnecessary escaping .. that happened automatically previously because of how jsonnet marshals yaml as text